config/config_mgmt.py

```diff
@@ -111,6 +113,136 @@ def readConfigDB(self):
 
         return
 
+    def writeConfigDB(self, jDiff):
+        print('Writing in Config DB')
+        """
+        On Sonic Switch
+        """
+        db_kwargs = dict(); data = dict()
+        configdb = ConfigDBConnector(**db_kwargs)
+        configdb.connect(False)
+        deep_update(data, FormatConverter.to_deserialized(jDiff))
+        data = sort_data(data)
+        self.logInFile("Write in DB: Last Data\n", data)
+        configdb.mod_config(FormatConverter.output_to_db(data))
+
+        return
+
+    """
+     Check ASIC DB whether interfaces present or not
+     ports: List of ports
+     portMap: port to OID map.
+     Return: True if all ports are not present.
+    """
+    def checkPortsInAsicDb(self, db, ports, portMap):
+
+        # internal funtions for better testing
+        def testAsicDB(oid):
+            # To Debug
+            if self.DEBUG_FILE:
+                cmd = 'sudo redis-cli -n 1 hgetall ' + \
+                '"ASIC_STATE:SAI_OBJECT_TYPE_PORT:osid:0x{}"'.format(oid)
+                print(cmd)
+                self.logInFile("Running {}".format(cmd))
+                system(cmd)
+            return
+
+        try:
+            # connect to ASIC DB,
+            db.connect(db.ASIC_DB)
+            oidKey = 'ASIC_STATE:SAI_OBJECT_TYPE_PORT:oid:0x*'
```

Comment:
This is just the prefix, why do we need the "*"?  We should define this as constant to be used by a few functions here.